streaming_data_types/arrays_wa00.py

```diff
@@ -0,0 +1,144 @@
+from typing import Union, Optional, NamedTuple
+from datetime import datetime, timezone
+from dataclasses import dataclass
+
+import flatbuffers
+import numpy as np
+from streaming_data_types.fbschemas.arrays_wav_00.WaveFormArray import WaveFormArray
+from streaming_data_types.fbschemas.arrays_wav_00.DType import DType
+from streaming_data_types.fbschemas.arrays_wav_00.WaveFormArray import (
+    WaveFormArrayStart,
+    WaveFormArrayAddXData,
+    WaveFormArrayAddYData,
+    WaveFormArrayAddXDataType,
+    WaveFormArrayAddYDataType,
+    WaveFormArrayAddYTimestamp,
+    WaveFormArrayAddXTimestamp,
+    WaveFormArrayAddYUnit,
+    WaveFormArrayAddXUnit,
+    WaveFormArrayAddNumberOfElements,
+    WaveFormArrayEnd,
+)
+from streaming_data_types.utils import check_schema_identifier
+
+FILE_IDENTIFIER = b"wa00"
+
+
+def serialise_wa00(
+    values_x_array: np.ndarray,
+    values_y_array: np.ndarray,
+    y_timestamp: Optional[datetime] = None,
+    x_timestamp: Optional[datetime] = None,
+    y_unit: Optional[str] = None,
+    x_unit: Optional[str] = None,
```

Comment:
You need to inject a little sanity into this argument list:
 - You don't usually want this many positional arguments, so a little `*,` to make keyword-only arguments
 - You change the x-y order part way through. IMO "x, y" is the correct order.

I'd add the `*,` after `values_y_array`. You don't technically need to re-order the x-y pairs of the later arguments, but it might look prettier.